src/server/main.cpp

```diff
@@ -144,6 +145,15 @@ int server_main(int, char **argv, char **) {
       fb.SendUpdate(rect, buf.xochitl_update.waveform,
                     buf.xochitl_update.flags);
     } break;
+    case ipc::WAIT_t: {
+      fb.WaitForLastUpdate();
```

Comment:
The wait for last update method does nothing IIRC

Reply:
i was wondering why this code works if fb.WaitForLastUpdate does nothing. i realize that technically doesn't matter whether it does something or not, since we go through the queue in order and all draw calls seem to be somewhat synchronous. but still, prefer to use the wait API they have to express our intent